src/routes/tools/local-climate-change-snapshot/_data.js

```diff
@@ -158,10 +166,15 @@ export async function getObserved(config, params, method = "GET") {
  * @param {string} method - default is GET, POST for uploaded boundaries
  * @return {array}
  */
-export async function getProjections(config, params, method = "GET") {
+export async function getProjections(
+  config,
+  params,
+  method = "GET",
```

Comment:
Ditto here for the `method` param and updating the doc string comment.

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/209/commits/a0404de4c1dba0b67f5b07113c31c051e6c2f015